stakedb/ticketpool.go

```diff
@@ -225,18 +175,20 @@ func NewTicketPool(dataDir, dbSubDir string) (tp *TicketPool, err error) {
 	// Open ticket pool diffs database
 	badgerDbPath := filepath.Join(dataDir, dbSubDir)
 	opts := badger.DefaultOptions(badgerDbPath)
+	opts.MetricsEnabled = false
 	opts.Logger = &badgerLogger{log}
 	db, err := badger.Open(opts)
-	if err == badger.ErrTruncateNeeded {
-		log.Warnf("NewTicketPool badger db: %v", err)
-		// Try again with value log truncation enabled.
-		opts.Truncate = true
-		log.Warnf("Attempting to reopening ticket pool db with the Truncate option set...")
-		db, err = badger.Open(opts)
-	}
 	if err != nil {
-		return nil, fmt.Errorf("failed badger.Open: %v", err)
+		if !strings.Contains(err.Error(), "manifest has unsupported version") {
```

Comment:
Wow, how annoying they didn't put this in an exported variable.

Reply:
Yeah.